packages/website/src/components/Layout/Header/index.tsx

```diff
@@ -17,15 +17,15 @@ const Header = (props: Props) => (
         text="GitHub"
       />
       <Button
-        icon="twitter"
+        icon="twitter-x"
         link={
           'https://twitter.com/intent/tweet?text=gitmoji' +
           '%20%E2%80%93%20An%20%23emoji%20guide%20for%20your%20commit' +
           '%20messages%20by%20%40crloscuesta%20%F0%9F%98%8D%F0%9F%98%9C' +
           '&url=https://gitmoji.dev'
         }
         target="_blank"
-        text="Tweet"
+        text="Post on X"
```

Comment:
I think `X` is redundant here, I would suggest either keeping:

- Tweet
- Share 

```suggestion
        text="Tweet"
```